src/herder/test/HerderTests.cpp

```diff
@@ -647,10 +647,6 @@ testTxSetWithFeeBumps(uint32 protocolVersion)
 
 TEST_CASE("txset", "[herder][txset]")
 {
-    SECTION("protocol 13")
```

Comment:
Yeah, we shouldn't need to create tx sets for protocol 13 anymore, right?

Reply:
we can't create new protocol 13 tx sets, but we need to be careful as to not accidentally remove replay test coverage (replay also uses TxSetFrame and the whole ledgerClose flow). That being said, I agree all tests covering surge pricing and block construction can be removed because replay just uses ready tx sets directly from the archives.

Reply:
Extra tests should have been removed in the cleanup PR, I only removed one helper here that is no longer used.